data_science_pipeline/utils/editor_recommendation.py

```diff
@@ -0,0 +1,12 @@
+from typing import Iterable
+import pandas as pd
+
+
+def get_author_ids_of_given_version_of_paper(
```

Comment:
minor naming: I would maybe keep the terminology consistent.
Within BigQuery it is called "Manuscript Version".
And so the data frame is also a `manuscript_version_df`.
(In a `manuscript_df` I wouldn't expect a `version_id` field)